Makefile

```diff
@@ -34,13 +40,14 @@ download-ext:
 clean-ext:
 	$(MAKE) -C src_ext distclean
 
-clean: fastclean
```

Comment:
It's written so as to not fail if ocp-build is not present… but yes, that's probably not the best approach.
`rm -rf _obuild/` (instead of `ocp-build -clean`) should actually be enough.